numpy/core/tests/test_simd.py

```diff
@@ -736,11 +736,14 @@ def test_arithmetic_div(self):
     def test_arithmetic_reduce_sum(self):
         """
         Test reduce sum intrinics:
+            npyv_sum_u8
+            npyv_sum_u16
             npyv_sum_u32
+            npyv_sum_u64
             npyv_sum_f32
             npyv_sum_f64
         """
-        if self.sfx not in ("u32", "f32", "f64"):
+        if self.sfx not in ("u8", "u16", "u32", "u64", "f32", "f64"):
```

Comment:
please add a separate test case for sumup covers multiple ranges of data range including the max ranges of integers to detect any unexpected behaviors such as saturations or overflow.
similar to the following:
```Python
def test_arithmetic_reduce_sumup(self):
        if self.sfx not in ("u8", "u16"):
            return
        rdata = (0, self.nlanes, self._int_min(), self._int_max()-self.nlanes)
        for r in rdata:
            data = self._data(r)
            vdata = self.load(data)
            data_sum = sum(data)
            vsum = self.sum(vdata)
            assert vsum == data_sum
```